src/reducers/cacheReducer.js

```diff
@@ -588,31 +588,7 @@ const initialize = (state, { action, key, path }) =>
     });
 
     // append docs field to query
-    reprocessQuerires(draft, path);
-
-    done();
-    return draft;
-  });
-
-const conclude = (state, { action, key, path }) =>
-  produce(state, (draft) => {
```

Comment:
It seems fine but I wonder if we want to make it a config instead? Either on the query (like we did here: https://github.com/TARAAI/redux-firestore/blob/master/src/actions/firestore.js#L270-L271) or on the global config (here: https://github.com/TARAAI/redux-firestore/blob/master/src/constants.js#L122-L134). That way if we do want to change it for some reason we could just enable/disable it without pushing a new build. Thoughts?